common/static/js/vendor/tinymce/js/tinymce/plugins/codemirror/source.html

```diff
@@ -30,13 +55,24 @@
 {// Initialise (before load)
 	"use strict";
 
-	tinymce = parent.tinymce;
-	editor = tinymce.activeEditor;
-    var i, userSettings = editor.settings.codemirror ? editor.settings.codemirror : {};
+    var i;
+    var getParamByName = function (name) {
+        var match = new RegExp('[?&]' + name + '=([^&]*)').exec(window.location.search);
+        return match && decodeURIComponent(match[1].replace(/\+/g, ' '));
+    };
+
+
+    // Getting this as a query parameter is very ugly, but the JS and CSS needs
+    // to be added to the head as the page is being rendered. Therefore it isn't possible
+    // to use postMessage APIs to get the setting value. And we can't hardcode the value in here
+    // because we need the baseURL, which is only added to server-side templated files.
+    var codemirrorPath = getParamByName("CodeMirrorPath"),
+        parentOrigin = getParamByName("ParentOrigin");
+
     CMsettings = {
-		path: userSettings.path || 'codemirror-4.8',
-    indentOnInit: userSettings.indentOnInit || false,
-    disableFilesMerge: userSettings.disableFilesMerge || false,
+	    path: codemirrorPath,
+        indentOnInit: false,
+        disableFilesMerge: false,
```

Comment:
Nit: it seems that we're removing the code for `disableFilesMerge`, so it might not be doing anything anymore.

Reply:
Removed this.